### ISSUE TITLE
Enhancement request

### ISSUE THREAD

**Issue by moumitahalder** [wcmarketplace/dc-woocommerce-multi-vendor#1019]
3) On the vendor store and in a product page sidebar I would like to have all of this as Vendor Info:
– Logo (already exists) Should link to the vendor store
– Name (already exists) Should link to the vendor store
– Address (already exists) Would like to show only country
– Short biography (this is the store description) Enable the admin to change the label of this for the vendor dashboard and frontend
– Rating (already exists)
– Number of sales
– Date of registration
– Social profiles (already exists)
– Link to the seller’s website

1)
On the vendor store, tabs Products, Reviews and Policies are separate pages. Policies and Reviews are just text and do not need a full page reload. At least, load them with Ajax. But, it would be best to have just regular tabs and everything on the same page.

2)
On the product page, the Vendor Info tab only has the Vendor Name and Store Description.
I would like to show a lot more, which I discuss in paragraph 3.
But, for a quick fix, you can update the plugin to display everything that is on the vendor store: Profile Image, Name, Rating, Country, Social, Follow and Store Description.

When adding this to the product page, please add tags and classes to these elements, so that they can be styled with CSS. Right now, the Vendor Description doesn’t have a tag or class. I assume that this should be set in the plugin’s php.
Also, the vendor store description doesn’t have the html and css markup that was added in the MCE editor. Please, fix this so that the text formatting and styling is preserved.
